### DISCUSSION TITLE
Add delay to tooltip

### DISCUSSION THREAD

**Discussion by DerTimonius** [saadeghi/daisyui#2012] [Ideas / feature requests]
Thanks for this great package 🙌 

I was just playing around with tooltips a little bit and am using it on a button that scales on hover. The transition takes 300ms, but the tooltip is displayed pretty much instantly and gets then moved around during the transition.

I think it would be great if there was an option to delay the display of the tooltip!

**Comment by Masstronaut** [comment]
This would be great! I've tried `after:duration-300` on the tooltip which didn't work. Would be helpful if the tooltip animation duration were configurable as a CSS var or via a tailwind class.

**Comment by saadeghi** [comment]
You can use `hover:after:delay-1000 hover:before:delay-1000` class names.  
`before` is for the text and `after` is for the arrow
